packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -755,51 +755,59 @@ class _SelectableTextState extends State<SelectableText>
-        cursorOffset: cursorOffset,
-        paintCursorAboveText: paintCursorAboveText,
-        backgroundCursorColor: CupertinoColors.inactiveGray,
-        enableInteractiveSelection: widget.enableInteractiveSelection,
-        magnifierConfiguration:
-            widget.magnifierConfiguration ?? TextMagnifier.adaptiveMagnifierConfiguration,
-        dragStartBehavior: widget.dragStartBehavior,
-        scrollPhysics: widget.scrollPhysics,
-        scrollBehavior: widget.scrollBehavior,
-        autofillHints: null,
-        contextMenuBuilder: widget.contextMenuBuilder,
-      ),
+    Widget editable = EditableText(
+      key: editableTextKey,
+      style: effectiveTextStyle,
+      readOnly: true,
+      toolbarOptions: widget.toolbarOptions,
+      textWidthBasis: widget.textWidthBasis ?? defaultTextStyle.textWidthBasis,
+      textHeightBehavior: widget.textHeightBehavior ?? defaultTextStyle.textHeightBehavior,
+      showSelectionHandles: _showSelectionHandles,
+      showCursor: widget.showCursor,
+      controller: _controller,
+      focusNode: focusNode,
+      strutStyle: widget.strutStyle ?? const StrutStyle(),
+      textAlign: widget.textAlign ?? defaultTextStyle.textAlign ?? TextAlign.start,
+      textDirection: widget.textDirection,
+      textScaler: effectiveScaler,
+      autofocus: widget.autofocus,
+      forceLine: false,
+      minLines: widget.minLines,
+      maxLines: widget.maxLines ?? defaultTextStyle.maxLines,
+      selectionColor: widget.selectionColor ?? selectionColor,
+      selectionControls: widget.selectionEnabled ? textSelectionControls : null,
+      onSelectionChanged: _handleSelectionChanged,
+      onSelectionHandleTapped: _handleSelectionHandleTapped,
+      rendererIgnoresPointer: true,
+      cursorWidth: widget.cursorWidth,
+      cursorHeight: widget.cursorHeight,
+      cursorRadius: cursorRadius,
+      cursorColor: cursorColor,
+      selectionHeightStyle: widget.selectionHeightStyle,
+      selectionWidthStyle: widget.selectionWidthStyle,
+      cursorOpacityAnimates: cursorOpacityAnimates,
+      cursorOffset: cursorOffset,
+      paintCursorAboveText: paintCursorAboveText,
+      backgroundCursorColor: CupertinoColors.inactiveGray,
+      enableInteractiveSelection: widget.enableInteractiveSelection,
+      magnifierConfiguration:
+          widget.magnifierConfiguration ?? TextMagnifier.adaptiveMagnifierConfiguration,
+      dragStartBehavior: widget.dragStartBehavior,
+      scrollPhysics: widget.scrollPhysics,
+      scrollBehavior: widget.scrollBehavior,
+      autofillHints: null,
+      contextMenuBuilder: widget.contextMenuBuilder,
     );
+    // TODO(Renzo-Olivares): Update this logic to inform paragraphs within
+    // this widgets text about paragraph spacing https://github.com/flutter/flutter/issues/177408.
+    final double? paragraphSpacing = MediaQuery.maybeParagraphSpacingOf(context);
+    if (paragraphSpacing != null && paragraphSpacing.abs() > precisionErrorTolerance) {
```

Comment:
why the `precisionErrorTolerance` check? The platform value may fluctuate?

Reply:
This is no longer here `paragraphSpacing` is now nullable by default.